package.json

```diff
@@ -103,8 +103,7 @@
     "eslint-plugin-import": "^2.31.0",
     "eslint-plugin-jest": "^28.10.0",
     "eslint-plugin-jest-formatting": "^3.1.0",
-    "eslint-plugin-n": "^17.15.1",
-    "eslint-plugin-node": "^11.1.0",
```

Comment:
Lint is failing with error `ESLint couldn't find the plugin "eslint-plugin-node".` 
Is this removal required? do we need to fix eslint?

Reply:
looks solved+

Reply:
Should be fixed now.